js/style/style_layer.js

```diff
@@ -169,6 +169,20 @@ StyleLayer.prototype = util.inherit(Evented, {
         }
     },
 
+    getPaintValueStopZoomLevels: function(name) {
+        var transition = this._paintTransitions[name];
+        if (transition) {
+            return transition.declaration.getStopZoomLevels();
+        } else {
+            return [];
+        }
+    },
+
+    getPaintInterpolationValue: function(name, zoom) {
```

Comment:
Ah! I see. I'm not picky about the name as long as you're consistent with the name of the shader parameter. I found `calculateInterpolationT` enlightening for that reason.